EventVisualisation/Detectors/src/DataInterpreterRND.cxx

```diff
@@ -0,0 +1,124 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    DataInterpreterRND.cxx
+/// \author  Jeremi Niedziela
+
+#include "EventVisualisationDetectors/DataInterpreterRND.h"
+
+#include "EventVisualisationBase/EventManager.h"
+#include "EventVisualisationDataConverter/MinimalisticEvent.h"
+#include "EventVisualisationBase/ConfigurationManager.h"
+#include "EventVisualisationBase/Track.h"
+
+#include <TEveManager.h>
+#include <TEveTrackPropagator.h>
+#include <TGListTree.h>
+
+#include <iostream>
+
+using namespace std;
+
+namespace o2  {
+namespace EventVisualisation {
+  
+DataInterpreterRND::DataInterpreterRND()
+{
+}
+
+DataInterpreterRND::~DataInterpreterRND()
+{
+}
+
+TEveElement* DataInterpreterRND::interpretDataForType(EventManager::EDataType type)
+{
+  int multiplicity = 500*((double)rand()/RAND_MAX)+100;
```

Comment:
I suggest adding a comment that this is a mockup, or does this have a real meaning?

Reply:
This class generates random events - it will be used as an example for detector-specific visualisation, but we may also use it to prepare demos when no other source of data is available.

Reply:
The motivation would be to avoid generating a temporary object. Probably the overhead is marginal. But if a function is called often such small things can add up. As for the string manipulation, the only good argument is to compare directly
```
getDataTypeName(typeid) == "something"
```
or do something directly on the returned object before using it for comparison. But in many cases you would anyhow use a new variable for manipulation. Depends what you want to do, the function doesn't seem to be used currently.